dockerfiles/default-configs/node.yaml

```diff
@@ -0,0 +1,72 @@
+# [time] is a concatenation of spans. Where each span is an integer number and a suffix.
+# E.g.: [1y 2M 4w 8d 16h 32min 64s 128ms 256us]
+# Supported suffixes:
+#  nsec, ns -- microseconds
+#  usec, us -- microseconds
+#  msec, ms -- milliseconds
+#  seconds, second, sec, s
+#  minutes, minute, min, m
+#  hours, hour, hr, h
+#  days, day, d
+#  weeks, week, w
+#  months, month, M -- defined as 30.44 days
+#  years, year, y -- defined as 365.25 days
+
+# [file] logger config file
+log_config: /configs/logger.yaml
+# [str] node name, required to match with cluster config
+name: local_node
+# [uint] min count of successful operations on replicas to consider operation successful
+quorum: 1
+# [time] timeout for every GRPC operation
+operation_timeout: 3sec
+# [time] interval for checking connections
+check_interval: 5000ms
+# [simple, quorum] simple - without checking status
+cluster_policy: quorum
+# type of the backend [in_memory, stub, pearl]
+backend_type: pearl
+# interval for checking for blobs cleanup
+cleanup_interval: 1h
+# soft limit for count of max blobs to remain in ram
+open_blobs_soft_limit: 2
+# hard limit for count of max blobs to remain in ram
+open_blobs_hard_limit: 10
+
+# used only for 'backend_type: pearl'
+pearl:
+  # optional, default = true, enables linux AIO
+  enable_aio: true
+  # path to logfile with info about disks states switches, default = "/tmp/bob_events.csv"
+  disks_events_logfile: "/tmp/bob_events.csv"
```

Comment:
Do not use `/tmp` for data. It is better to use `/bob/events/` or `/bob/log/` here